plugins/woocommerce/client/admin/client/settings-payments/onboarding/providers/woopayments/steps/payment-methods-selection/payment-methods-selection.scss

```diff
@@ -0,0 +1,670 @@
+.woocommerce-embed-page {
+	#wpbody .woocommerce-layout,
+	.woocommerce-layout__notice-list-hide + .wrap {
+		padding-top: 0;
+	}
+}
+
+.settings-payments-main__container {
```

Comment:
What’s the purpose of applying the NOX page styles on the payment methods selection step?

Reply:
Thanks for the catch! This was a leftover from the copy/paste of the plugin and CSS files. Removed in b641937

Reply:
I believe there are more leftovers like:

Reply:
Thanks! Removed in https://github.com/woocommerce/woocommerce/pull/56554/commits/a7c1c87f1358fbbb5ac66206c58e541105037d93. I'll take another look to see if there are some extra things there.

That being said, and given these files were removed from their prior location, is there a risk this will break the UI? I'm not familiar with the earlier work, so wanted to check

Reply:
I didn’t find any removals of these styles in the PR changes, and the NOX page is displaying correctly, so we should be good. :)